lib/local-storage.coffee

```diff
@@ -14,14 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ###
 
-settings = require('resin-settings-client')
-
-# Fallback to filesystem based
-# storage if not in the browser.
-unless localStorage?
-	LocalStorage = require('node-localstorage').LocalStorage
-
+if typeof localStorage isnt 'undefined'
```

Comment:
Well, it's the same, but read to me a bit strange, as it references the potentially undefined variable. I know that actually CS does the same check, so it's only the matter of preference.